docs/api/structures/cookie.md

```diff
@@ -12,3 +12,4 @@
 * `expirationDate` Double (optional) - The expiration date of the cookie as
   the number of seconds since the UNIX epoch. Not provided for session
   cookies.
+* `sameSite` String - The [Same Site](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#SameSite_cookies) policy applied to this cookie.  Can be `unspecified`, `no_restriction`, `lax`, `strict` or `unknown`.
```

Comment:
What does `unknown` mean here?

Reply:
It means we didn't update the converter :/

c++ isn't smart enough to know I covered every enum so we need a default case in the converter.  We have this `unknown` value in a few places where we convert from an enum